ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -59,7 +59,8 @@ class UnityEnvironment(BaseEnv):
     # Revision history:
     #  * 1.0.0 - initial version
     #  * 1.1.0 - support concatenated PNGs for compressed observations.
-    API_VERSION = "1.1.0"
+    #  * 1.2.0 - support compression mapping for stacked compressed observations.
+    API_VERSION = "1.2.0"
```

Comment:
You'll need to update this method 
https://github.com/Unity-Technologies/ml-agents/blob/master/ml-agents-envs/mlagents_envs/environment.py#L117-L121
to make sure the flag is set.